cmd/influx/pkg.go

```diff
@@ -428,16 +430,21 @@ func (b *cmdPkgBuilder) readPkg() (*pkger.Pkg, bool, error) {
 	}
 	pkgs = append(pkgs, urlPkgs...)
 
+	// the pkger.ValidSkipParseError option allows our server to be the one to validate the
+	// the pkg is accurate. If a user has an older version of the UI and our cloud gets updated
```

Comment:
By "older version of the UI" do you mean CLI, or do you mean the future UI component?

Reply:
whoops, I did indeed 😬, great catch 👍

Reply:
fix pushed 👍